tests/src/Kernel/Entity/GetBundleByBundleTest.php

```diff
@@ -0,0 +1,557 @@
+<?php
+
+namespace Drupal\Tests\og\Kernel\Entity;
+
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\block_content\Entity\BlockContentType;
+use Drupal\node\Entity\NodeType;
+use Drupal\og\Og;
+use Drupal\og\OgGroupAudienceHelper;
+
+/**
+ * Tests retrieving group (content) bundles by group (content) bundles.
+ *
+ * @group og
+ * @coversDefaultClass \Drupal\og\GroupManager
+ */
+class GetBundleByBundleTest extends KernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'block_content',
+    'field',
+    'node',
+    'og',
+    'system',
+    'user',
+  ];
+
+  /**
+   * Test groups.
+   *
+   * @var \Drupal\Core\Entity\EntityInterface[][]
+   */
+  protected $groups = [];
+
+  /**
+   * Test group content.
+   *
+   * @var \Drupal\Core\Entity\EntityInterface
+   */
+  protected $groupContent;
+
+  /**
+   * The group manager.
+   *
+   * @var \Drupal\og\GroupManager
+   */
+  protected $groupManager;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    $this->installConfig(['og']);
+    $this->installEntitySchema('block_content');
+    $this->installEntitySchema('node');
+    $this->installEntitySchema('og_membership');
+    $this->installEntitySchema('user');
+    $this->installSchema('system', 'sequences');
+
+    $this->groupManager = $this->container->get('og.group.manager');
+
+    // Create four groups of two different entity types.
+    for ($i = 0; $i < 2; $i++) {
+      $bundle = "group_$i";
+      NodeType::create([
```

Comment:
actually, lets avoid `Node`, and use `EntityTest` instead -- like this we make extra sure there's no node specific code.